specification/sql/resource-manager/Microsoft.Sql/preview/2022-08-01-preview/examples/ElasticPoolCreateOrUpdateMin.json

```diff
@@ -11,6 +11,9 @@
   },
   "responses": {
     "200": {
+      "headers": {
```

Comment:
response 200 doesn't need to return `Location` header.

Reply:
Gotcha, removing response headers from response 200 in all example files. Thanks

Reply:
Added a new iteration by removing the headers from 200 response codes in all the example files. All checks validated